### DISCUSSION TITLE
deCONZ e config.txt

### DISCUSSION THREAD

**Discussion by calz64** [indomus/forum#865] [HUB: Home Assistant]
Qualcuno sa dirmi come si raggiunge questo file, leggo dalla documentazione di Home Assistant Add-on: deCONZ:
_____
If you are using RaspBee, you may need to edit the config.txt file in the root of your SD card in order for your RaspBee to be recognized and assigned a device name.
Add following to your config.txt:
```
enable_uart=1
dtoverlay=pi3-miniuart-bt
```
_____

deconz non va e credo vada riconfigurato, grazie

**Comment by KC-inDomus** [comment]
https://indomus.it/guide/come-installare-e-configurare-deconz-su-home-assistant-hassio/#installazioneantenna

**Comment by calz64** [comment]
In realtà quello che non funziona è l'integrazione ZigBee Home Automation che non si connette più alla chiavetta, mi da errore nel log trovo quello che ho riportato sopra. 
Il servizio Phoscon invece funziona e integra i sensori, anche se ha grossi limiti.

**Comment by calz64** [comment]
Ho capito che il problema, della mancata connessione dell'integrazione ZigBee Home Automation, era la chiavetta Conbee inserita direttamente nel raspy (ammetto di aver fatto confusione fra raspBee e ConBee, io ho la seconda), ho acquistato una prolunga (10 cm) ed è tornato a funzionare.
Avrei un dubbio: si può collegare un sensore ad entrambe i servizi zigbee > in questo caso ""Phoscon e "Zigbee Home Automation"?
